packages/uniorg-parse/src/parser.ts

```diff
@@ -425,7 +426,10 @@ class Parser {
       if (objectBegin !== prevEnd) {
         // parse text before object
         const value = this.r.substring(prevEnd, objectBegin);
-        objects.push(u('text', { value }));
+        const end = prevEnd + value.length;
+        objects.push(
+          u('text', { value, position: this.getPosition(prevEnd, end) })
```

Comment:
When `positions` setting is false, this assigns `undefined` to `position`. This is not valid for Unist because it requires AST to be JSON-serializable.

One option to fix that easily is like this:
```ts
getPosition(begin, end): { position?: Position } => {
  ...
}
```
and then
```ts
u('text', { value, ...this.getPosition(prevEnd, end) })
```